Day5/counting_ex7/Counter.java

```
@@ -0,0 +1,40 @@
+package counting_ex7;
+
+public class Counter {
+	
+	int count = 1;
+	synchronized  void countNumbers()  {
+		for(;count<=100;count++) {
+			System.out.println(count);
+			try {
+				Thread.sleep(1000);
+			}catch(Exception e) {}
+			if(count%10==0) {
+				try {
+					notify();
+					wait();
+				}catch(Exception e) {}
```

@NachiappanAnandan  Avoid having an empty catch block, Try to have some useful information logged there, like print stack trace. (Applicable for all empty catch blocks used, change all empty catch blocks)

cc: @jenisha-roshan